app/editor_ui/templates/editor_ui/projects/project_detail.html

```diff
@@ -58,6 +58,17 @@ <h2 class="tna-heading-m">
             <span class="tna-heading-xl u-pl-s">{{ responses_count }}</span><a href="{{ url("editor_ui:projects:responses:list", project_uuid=project.uuid) }}">response{{ responses_count|pluralize }}</a>
         </div>
       </div>
+
+      <div class="tna-column tna-column--width-1-3 tna-column--full-small tna-column--full-tiny tna-!--margin-top-l">
```

Comment:
This should match the design style of the `Create feedback form` button. 
I've just kept this as a placeholder for now.